foundations/foundation-metrics/src/main/java/io/servicecomb/foundation/metrics/MetricsObserverManager.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicecomb.foundation.metrics;
+
+import java.io.File;
+import java.util.concurrent.TimeUnit;
+
+import org.springframework.stereotype.Component;
+
+import com.netflix.config.DynamicPropertyFactory;
+import com.netflix.servo.publish.BasicMetricFilter;
+import com.netflix.servo.publish.CounterToRateMetricTransform;
+import com.netflix.servo.publish.FileMetricObserver;
+import com.netflix.servo.publish.MemoryMetricObserver;
+import com.netflix.servo.publish.MetricObserver;
+import com.netflix.servo.publish.MonitorRegistryMetricPoller;
+import com.netflix.servo.publish.PollRunnable;
+import com.netflix.servo.publish.PollScheduler;
+
+@Component
+public class MetricsObserverManager {
+
+  protected static final String METRICS_POLL_TIME = "cse.metrics.polltime";
+
+  protected static final String MEM_ENABLED = "cse.metrics.mem.enabled";
+
+  protected static final String FILE_ENABLED = "cse.metrics.file.enabled";
+
+  private static final String FILE_NAME = "cse.metrics.file.name";
+
+  private static final String FILE_PATH = "cse.metrics.file.path";
+
+  private int metricPoll = 60;
+
+  private MemoryMetricObserver memoryObserver = null;
```

Comment:
the memoryObserver field is initiated with null by default, so you don't need to asgin the null here.